_health-care/_js/components/Nav.jsx

```diff
@@ -13,7 +13,9 @@ class Nav extends React.Component {
           </Link>
         </li>
         <li role="presentation">
-          Personal Information
+          <Link to="/personal-information" className={/personal-information/.test(this.props.currentUrl) ? ' usa-current' : ''}>
```

Comment:
This regexp doesn't quite work right.  It'd match for `/military-service/some-personal-information-here/`.

Instead, try `this.props.location.pathname.startsWith('/personal-information/')`